packager/media/formats/mp4/box_definitions.cc

```diff
@@ -1020,7 +1020,7 @@ SampleGroupDescription::SampleGroupDescription() = default;
 SampleGroupDescription::~SampleGroupDescription() = default;
 
 FourCC SampleGroupDescription::BoxType() const {
-  return FOURCC_sgpd;
+  return FOURCC_senc;
```

Comment:
That is incorrect way to address the problem as the box type cannot be "senc". It will cause playback problems.